### DISCUSSION TITLE
How to clear the initially selected row?

### DISCUSSION THREAD

**Discussion by ptheofan** [bosskmk/pluto_grid#801] [Q&A]
<img width="484" alt="image" src="https://user-images.githubusercontent.com/880939/227665730-70ff45bc-3a3e-45b9-8e5f-68ac0a6b7418.png">

When the grid starts it has the first row selected. This "selection" normally clears when I press the ESC key.

**I would like it to start with none selected.**

I have tried the following with no luck
```dart
event.stateManager.clearCurrentSelecting(notify: true);
event.stateManager.setCurrentCell(null, null);
```

**Comment by darkomenx** [comment]
For my I started with empty value in select column BUT if I fill column then it impossible to reset field to empty.

The only solution found is added an empty option string in my list.
